crates/topos-tce-broadcast/src/tests/mod.rs

```diff
@@ -38,29 +33,17 @@ fn small_config() -> TceParams {
 fn medium_config() -> TceParams {
     TceParams {
         nb_peers: 50,
-        echo_subscribers_sample_size: 5,
```

Comment:
Maybe we need to update the threshold regarding the new size, from 20 to 50. The values can be found in Notion.

Reply:
@gruberb you still need to update the different threshold regarding the comment above